core/assets/currencies.go

```diff
@@ -5,13 +5,23 @@ import (
 	"database/sql/driver"
 	"errors"
 	"fmt"
+	"math"
 	"math/big"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/willf/pad"
 )
 
 var ErrNoQuotesForCurrency = errors.New("cannot unmarshal json.Number into currency")
+var wei *big.Float
+var ok bool
+
+func init() {
+	wei, ok = new(big.Float).SetString("1000000000000000000")
```

Comment:
What do you think about placing this in our Ethereum package?
Side Note: you can use separators in upcoming golang versions, for instance 1_000_000